packages/main/src/themes/base/ListItem.less

```diff
@@ -57,6 +57,11 @@
 		-webkit-box-align: center;
 		align-items: center;
 	}
+
+	.deleteListItem {
```

Comment:
The name is a bit confusing, I get it as these styles are applied to the list item itself (rather on the delete action wrapper element) and we should follow the css naming convention for this file. As the new DIV is rendered as pear to sapMLIBContent, you can name it something like sapMLIBContentSelection